content/en/llm_observability/sdk.md

```diff
@@ -104,6 +104,18 @@ LLMObs.enable(
 : optional - _string_
 <br />The name of the service used for your application. If not provided, this defaults to the value of `DD_SERVICE`.
 
+### Serverless setup
+
+Enable LLM Observability in a serverless environmant by using the [Datadog-Python and Datadog-Extension layers][14].
```

Comment:
Maybe update the wording to specify exactly what "using" means, e.g. the particular actions that need to be done with relation to the layers to enable LLM observability. Otherwise, a user might jump to server-less docs and get distracted there since they don't have clear action items after navigating to that page.

e.g.
"follow setup instructions for Datadog-Extension layers and LLM observability will be automatically enabled .."?

Reply:
that would be a lot better, i'll change the wording to something like that!

Reply:
updated. it's a bit tricky, since the steps in the Datadog AWS Lambda documentation are just _steps_, and I can't link to an individual step to reference setting up Datadog-Extension and Datadog-Python. however, i did update the language basically to what you suggested, lmk what you think!